spec/models/topic_spec.rb

```diff
@@ -77,126 +77,126 @@
 
-        user.update!(trust_level: 0)
+      user.update!(trust_level: 0)
 
-        expect(Topic.qa_can_vote(topic, user)).to eq(false)
-      end
+      expect(Topic.qa_can_vote(topic, user)).to eq(false)
+    end
 
-      it 'return false if has voted more than qa_tl*_vote_limit' do
-        SiteSetting.qa_trust_level_vote_limits = true
+    it 'return false if has voted more than qa_tl*_vote_limit' do
+      SiteSetting.qa_trust_level_vote_limits = true
 
-        expect(Topic.qa_can_vote(topic, user)).to eq(true)
+      expect(Topic.qa_can_vote(topic, user)).to eq(true)
 
-        SiteSetting.send("qa_tl#{user.trust_level}_vote_limit=", 1)
+      SiteSetting.send("qa_tl#{user.trust_level}_vote_limit=", 1)
 
-        QuestionAnswer::VoteManager.vote(answers[0], user, direction: up)
+      QuestionAnswer::VoteManager.vote(answers[0], user, direction: up)
 
-        expect(Topic.qa_can_vote(topic, user)).to eq(false)
+      expect(Topic.qa_can_vote(topic, user)).to eq(false)
 
-        SiteSetting.send("qa_tl#{user.trust_level}_vote_limit=", 2)
+      SiteSetting.send("qa_tl#{user.trust_level}_vote_limit=", 2)
 
-        expect(Topic.qa_can_vote(topic, user)).to eq(true)
-      end
+      expect(Topic.qa_can_vote(topic, user)).to eq(true)
     end
+  end
 
-    describe '#qa_votes' do
-      it 'should return nil if user is blank' do
-        expect(Topic.qa_votes(topic, nil)).to eq(nil)
-      end
+  describe '.qa_votes' do
+    it 'should return nil if user is blank' do
+      expect(Topic.qa_votes(topic, nil)).to eq(nil)
+    end
 
-      it 'should return nil if disabled' do
-        SiteSetting.qa_enabled = false
+    it 'should return nil if disabled' do
+      SiteSetting.qa_enabled = false
 
-        expect(Topic.qa_votes(topic, user)).to eq(nil)
-      end
+      expect(Topic.qa_votes(topic, user)).to eq(nil)
+    end
 
-      it 'should return voted post IDs' do
-        expected = answers.first(3).map do |a|
-          QuestionAnswer::VoteManager.vote(a, user, direction: up)
+    it 'should return voted post IDs' do
+      expected = answers.first(3).map do |a|
+        QuestionAnswer::VoteManager.vote(a, user, direction: up)
 
-          a.id
-        end.sort
+        a.id
+      end.sort
 
-        expect(Topic.qa_votes(topic, user).pluck(:post_id))
-          .to contain_exactly(*expected)
-      end
+      expect(Topic.qa_votes(topic, user).pluck(:post_id))
+        .to contain_exactly(*expected)
     end
+  end
 
-    describe '#qa_enabled' do
-      it 'should return false if topic is blank' do
-        expect(Topic.qa_enabled(nil)).to eq(false)
-      end
+  describe '.qa_enabled' do
+    it 'should return false if topic is blank' do
+      expect(Topic.qa_enabled(nil)).to eq(false)
+    end
 
-      it 'should return false if disabled' do
-        SiteSetting.qa_enabled = false
+    it 'should return false for a PM' do
+      expect(Topic.qa_enabled(Fabricate(:private_message_topic))).to eq(false)
```

Comment:
This is the test I added among all the indentation changes.

Reply:
Really don't understand why GitHub doesn't just default to not showing whitespace changes. Totally irrelevant in most cases.